### PR TITLE
Initial release

### PR DESCRIPTION
**Pull request by pvdlg**
Ready to go once semantic-release/semantic-release#578 lands.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,75 @@
...
+
+## Configuration
+
+### Atom installation
+
+The `apm` command line has to be installed in your CI environment and available in the `PATH`.
+
+See the [Atom Package CI Scripts](https://github.com/atom/ci#atom-package-ci-scripts) documentation.
+
+### Github authentication
```
Comment (shroudedcode) [inline README.md]: *GitHub* should be correctly capitalized everywhere.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,75 @@
...
+### Environment variables
+
+| Variable                     | Description                                                          |
+|------------------------------|----------------------------------------------------------------------|
+| `GH_TOKEN` or `GITHUB_TOKEN` | **Required.** The token used to authenticate with GitHub repository. |
+| `ATOM_ACCESS_TOKEN`          | **Required.** The token used to authenticate with Atom registry.     |
+
+### Additional options
+
+This shareable config uses the [git](https://github.com/semantic-release/git), [npm](https://github.com/semantic-release/npm), [exec](https://github.com/semantic-release/exec), [changelog](https://github.com/semantic-release/changelog) and [github](https://github.com/semantic-release/github) plugins. See the documentation of each plugins for additional options.
```
Comment (shroudedcode) [inline README.md]: Maybe format the plugin names as `code`, just like `semantic-release`.

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  - 8
+
+services:
+  - docker
+
+addons:
+  apt:
+    packages:
+    # See https://github.com/atom/ci
+    - libsecret-1-dev
```
Comment (shroudedcode) [inline .travis.yml]: Sorry for the nitpickyness, but this should be indented.

File: README.md (hunk 1 above)
Comment (pvdlg) [reply README.md]: For some reasons I was convinced that *Github* was the correct capitalization, but you're right it's * GitHub* indeed.

It's fixed now

File: README.md (hunk 2 above)
Comment (pvdlg) [reply README.md]: Done!

File: .travis.yml (hunk 3 above)
Comment (pvdlg) [reply .travis.yml]: Done!

**Comment by pvdlg** [comment]
Thanks for the review @shroudedcode ! I did the changes.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,75 @@
...
+
+### Atom installation
+
+The `apm` command line has to be installed in your CI environment and available in the `PATH`.
+
+See the [Atom Package CI Scripts](https://github.com/atom/ci#atom-package-ci-scripts) documentation.
+
+### GitHub authentication
+
+The `GitHub` authentication configuration is **required** and can be set via [environment variables](#environment-variables).
```
[the remark this answers is not in the document]
Comment (pvdlg) [reply README.md]: Done

File: README.md [hunk 5]
```diff
@@ -1,2 +1,75 @@
...
+
+Follow the [Creating a personal access token for the command line](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line) to obtain an authentication token. The token has to be made available in your CI environment via the `ATOM_ACCESS_TOKEN` environment variable.
+
+### Atom authentication
+
+The `Atom` authentication configuration is **required** and can be set via [environment variables](#environment-variables).
```
[the remark this answers is not in the document]
Comment (pvdlg) [reply README.md]: and done

File: README.md (hunk 2 above)
Comment (pvdlg) [reply README.md]: I really like your [set of conventions on when to use code formatting](https://gist.github.com/shroudedcode/4b889af9e82a8f2b3c893b7cd03a6390).

Here `semantic-release` is a package name while the plugin name are not the package name. So not sure they should be `code` then...

**Pull request merged**